src/modules/Client/Api/Guest.php

```diff
@@ -84,10 +84,10 @@ public function create($data = [])
         $this->getService()->checkExtraRequiredFields($data);
         $this->getService()->checkCustomFields($data);
 
-        $this->di['validator']->isPasswordStrong($this->di['array_get']($data, 'password'));
+        $this->di['validator']->isPasswordStrong($data['password']);
```

Comment:
Should `$data['password']` not default to `null` when no value is provided?

Reply:
Nope, it should be fine without it. The password is one of the values that are marked as required, which is checked on [line 78](https://github.com/FOSSBilling/FOSSBilling/blob/main/src/modules/Client/Api/Guest.php#L78), so the function should never reach this point if it's not provided